### PR TITLE
Fixes #2119 #2118 

### LINKED ISSUE #2119: Request: Horizontal layouts for all screens
**Actual Behaviour**

At the moment not all screens have an extra layout for horizontal (landscape) orientation. Instead the screens are displayed in vertical (portrait) orientation, no matter of the orientation of the device.

**Expected Behaviour**

It would be really nice to have horizontal layouts for all screens in order to improve user experience for devices which can be placed on a table in horizontal orientation.

**Steps to reproduce it**

Open app, choose multimeter

**Photo of the issue**

IMG_20200610_225851

**Would you like to work on the issue?**

Not really.

### PR DESCRIPTION
**Pull request by riturajjain2000**
Fixes #2119 #2118 

**Changes**: Added Tablet layout for multimeter , unlocked landscape mode for some activities and fixed landscape mode of Splash Screen
Had to add some values in `dimens.xml` as Tablet layout need more space than normal one.

**Screenshot/s for the changes**: 
Android Emulator - Pixel_C_API_29_5556 05-10-2020 20_29_41
Android Emulator - Pixel_C_API_29_5556 05-10-2020 20_28_59
Android Emulator - Pixel_C_API_29_5556 05-10-2020 20_29_34
Android Emulator - Pixel_C_API_29_5556 05-10-2020 21_27_51

**Checklist**:
- [x] I have used resources from `strings.xml`, `dimens.xml` and `colors.xml` without hard-coding them
- [x] No modifications done at the end of resource files `strings.xml`, `dimens.xml` or `colors.xml`
- [x] I have reformatted code in every file included in this PR [<kbd>CTRL</kbd>+<kbd>ALT</kbd>+<kbd>L</kbd>]
- [x] My code does not contain any extra lines or extra spaces
- [x] I have requested reviews from other members

**APK for Testing**:
[tablet_layout-and-landscape_splash_screen.zip](https://github.com/fossasia/pslab-android/files/5348434/tablet_layout-and-landscape_splash_screen.zip)
[Made_changes asked.zip](https://github.com/fossasia/pslab-android/files/5348435/Made_changes.asked.zip)

### CODE REVIEW

**Review by vsasvipul0605 — APPROVED** [review]

**Review by CloudyPadmal — CHANGES_REQUESTED** [review]
The AndroidManifest changes would affect the phone layout. Don't change the orientation there.

File: app/src/main/AndroidManifest.xml [hunk 1]
```diff
@@ -73,21 +76,21 @@
...
-            android:screenOrientation="portrait"
+            android:screenOrientation="user"
             android:windowSoftInputMode="stateHidden|adjustResize" />
         <activity
             android:name=".activity.LuxMeterActivity"
             android:screenOrientation="portrait" />
         <activity
             android:name=".activity.WaveGeneratorActivity"
-            android:screenOrientation="portrait" />
+            android:screenOrientation="user" />
```
Comment (CloudyPadmal) [inline app/src/main/AndroidManifest.xml:93]: Revert back and use a layout file for tabs. Refer to comments for a screenshot.

File: app/src/main/AndroidManifest.xml [hunk 2]
```diff
@@ -73,21 +76,21 @@
...
+            android:screenOrientation="sensorLandscape" />
         <activity android:name=".activity.SensorDataLoggerActivity" />
         <activity
             android:name=".activity.MultimeterActivity"
-            android:screenOrientation="portrait" />
+            android:screenOrientation="user" />
         <activity
             android:name=".activity.PowerSourceActivity"
-            android:screenOrientation="portrait"
+            android:screenOrientation="user"
```
Comment (CloudyPadmal) [inline app/src/main/AndroidManifest.xml:86]: Revert back and use a layout file for tabs. Refer to comments for a screenshot.

**Review by CloudyPadmal — CHANGES_REQUESTED** [review]

File: app/src/main/AndroidManifest.xml (hunk 1 above)
Comment (riturajjain2000) [reply app/src/main/AndroidManifest.xml:93]: Ok sir , will do that

**Review by CloudyPadmal — APPROVED** [review]
Thank you for the PR! :+1:

**Comment by riturajjain2000** [comment]
> Thank you for the PR! 👍

Happy to help sir :)

**Pull request merged**